i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,33 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+### Verhalten der Eingabetaste ändern
+
+In diesem Abschnitt können Sie die Aktion „Drücken Sie die Eingabetaste“ ändern. Diese Einstellung gilt pro Gerät. Nach der Abmeldung wird es wieder auf die Standardaktion zurückgesetzt.
```

Comment:
Same as for the English version, would suggest:
```
In diesem Abschnitt finden Sie eine Anleitung, wie Sie das Verhalten der Eingabetaste verändern können.

:::info
Diese Einstellung kann für jedes Gerät einzeln vorgenommen werden und wird bei der Abmeldung auf die Standardeinstellung zurückgesetzt. Sie müssen sie nach jeder Anmeldung wieder neu setzen.
:::
```

Reply:
Fixed :)